unittests/CppInterOp/FunctionReflectionTest.cpp

```diff
@@ -532,6 +532,9 @@ TEST(FunctionReflectionTest, ExistsFunctionTemplate) {
 }
 
 TEST(FunctionReflectionTest, InstantiateTemplateFunctionFromString) {
+#ifdef EMSCRIPTEN
+  GTEST_SKIP() << "Test crashes gtest on Emscipten";
+#endif
```

Comment:
I see this also returns a non nullptr so technically EXPECT_TRUE should work here 

![image](https://github.com/user-attachments/assets/5b9735e9-6c1d-49a1-b04c-abb6aef62bc2)